README.md

```diff
@@ -76,45 +76,27 @@ Prerequisites to Run Integration Tests
-    
-    * https://github.com/openshift/origin/blob/master/CONTRIBUTING.adoc
-    
-    You may take a binary built by the OpenShift team, build an instance locally, obtain through the CDK, or use Vagrant; any way that boots an `upstream/master` version of OpenShift locally should be fine.
-    
-    When running, this should give you a local API to execute against at https://localhost:8443, which is where the OpenShiftService tests currently look to make their calls (by default).  To override this, you may specify the environment variable or system property `KONTINUITY_CATAPULT_OPENSHIFT_URL`.
-    
-4. Configure OpenShift
-
-    For the time being, we need to do a little configuration in OpenShift to add the support we depend upon.
-    
-    * Log into OpenShift
-        * From ADB, this is done after `vagrant up` has completed:
-            * `vagrant ssh`
-        * `oc login`
-            * Use `admin`/`admin` as the username/password
-        * Add Jenkins Template Support
-            * When Using ADB: `oc create -f https://raw.githubusercontent.com/openshift/origin/master/examples/jenkins/pipeline/jenkinstemplate.json -n openshift`
-            * When Building Locally : `oc create --config=openshift.local.config/master/admin.kubeconfig -f https://raw.githubusercontent.com/openshift/origin/master/examples/jenkins/pipeline/jenkinstemplate.json -n openshift`            
-
-5. Configure Local Environment
-
-    By default, the Catapult console will display known pipelinable applications as read from the application store repository located at:
-    
-      `https://raw.githubusercontent.com/redhat-kontinuity/appstore/master/appstore.json`
-      
-    To override this default repository location, set the environment variable or the system property `CATAPULT_APPSTORE_URL`.  If both are set, the system property will take precedence. 
+    * Install minishift and prerequisite projects by following these instructions
+        * https://github.com/minishift/minishift#installing-minishift
+	
+    * Check everything works okay by loggin in to the OpenShift console
+        * Run `minishift start`
+        * Open the URL found in the output of the previous command in a browser. You can get the same URL by executing `minishift console --url` as well.
+        * Log in with user `admin` and password `admin`
+        * You may have to accept some security exceptions in your browser because of missing SSL Certificates
+
+    * Set up the following environment variables:
+        ```
+        CATAPULT_OPENSHIFT_API_URL=<insert minishift console url something like https://192.168.99.128:8443>
+        CATAPULT_OPENSHIFT_CONSOLE_URL=<insert minishift console url something like https://192.168.99.128:8443>
+        ```
+        
+        You can do this automatically in the following way:
+        
+        ```
+        export CATAPULT_OPENSHIFT_API_URL=`minishift console --url`
```

Comment:
It seems to work even when MiniShift is stopped so I guess so. Personally I tend not to put project-specific things in the bash startup scripts because you never know how they might interfere with each other. So I put those things an a `setenv.sh` script that I keep in the root of my `projects/obsidian` folder.